src/utils/PlaygroundApp.js

```diff
@@ -0,0 +1,47 @@
+import { FSHTank, RawFSH } from 'fsh-sushi/dist/import';
+import { exportFHIR } from 'fsh-sushi/dist/export';
+import { logger, Type } from 'fsh-sushi/dist/utils';
+import { FHIRDefinitions } from 'fsh-sushi/dist/fhirdefs';
+import { Configuration } from 'fsh-sushi/dist/fshtypes';
```

Comment:
I'm not entirely sure why the linter didn't pick this one up, but I don't think `Configuration` is ever used, so I don't think you need this import.

Reply:
Oh wait I just noticed that we do set `let config = Configuration` below, but I don't think that's exactly right. `Configuration` is a type in SUSHI, and we can't really set a variable to be the type, so I don't think the initial setting of `config` is necessary and we can just wait for it to be set when we run `readConfigPlayground` and then the import isn't needed.